src/pages/ValidateConfigFiles.js

```diff
@@ -185,44 +176,18 @@ class ValidateConfigFiles extends Component {
 
   validateModel = () => {
     this.setState({valid: VALIDATING, invalidMsg: ''});
-    // for testing purposes, set dev = true
-    // to switch between valid and invalid results
-    var dev = false;
-    var bodyStr = '';
-    if (dev) {
-      bodyStr = (this.state.valid !== INVALID) ? {'want_fail': true} : {'want_pass': true};
-    }
 
-    fetch(getAppConfig('shimurl') + '/api/v1/clm/config_processor', {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify(bodyStr)
-    })
-      .then(response => {
-        if (response.ok) {
-          this.setState({valid: VALID}, () => {
-            this.props.enableNextButton(true);
-          });
-          this.clearAllChangeMarkers();
-          return JSON.stringify('');  // success call do not return any json
-        } else {
-          this.setState({valid: INVALID}, () => {
-            this.props.enableNextButton(false);
-          });
-          return response.json();
-        }
+    postJson('/api/v1/clm/config_processor')
+      .then(() => {
+        this.setState({valid: VALID}, () => {
+          this.props.enableNextButton(true);
+        });
+        this.clearAllChangeMarkers();
       })
-      .then(responseData => {
-        if (responseData.log) {
-          if (dev) {
-            var msg = 'ERR: Server role \'HLM-ROLE2\' used by server deployer is not defined';
-            this.setState({invalidMsg: msg});
-          } else {
-            this.setState({invalidMsg: responseData.log});
-          }
-        }
+      .catch(resp => {
+        this.props.enableNextButton(false);
+        resp.json().then(reason =>
+          this.setState({valid: INVALID, invalidMsg: reason.log}));
       });
   }
```

Comment:
I got some errors when I run validate error from backend
 
INFO:ardana_service:2017-11-28 18:56:04.223087 127.0.0.1     POST http://localhost:8081/api/v2/config_processor
ERROR:ardana_service.config_processor:Command '['/home/gugl/dev/day0React/ardana-service/tools/mock-cmd', '/home/gugl/dev/day0React/ardana-service/data/config-processor/bin/python', '/home/gugl/dev/day0React/ardana-service/data/config-processor/ardana-configuration-processor/Driver/ardana-cp', '-l', '/tmp/tmpFWKCXd/log', '-c', '/home/gugl/dev/day0React/ardana-service/data/my_cloud/model/cloudConfig.yml', '-s', '/home/gugl/dev/day0React/ardana-service/data/ardana-input-model/2.0/services', '-r', '/home/gugl/dev/day0React/ardana-service/data/config-processor/ardana-configuration-processor/Data/Site', '-o', '/tmp/tmpFWKCXd/clouds']' returned non-zero exit status 254
Traceback (most recent call last):
  File "/home/gugl/dev/day0React/ardana-service/ardana_service/config_processor.py", line 93, in run_config_processor
    subprocess.check_output(cmd, stderr=subprocess.STDOUT)
  File "/usr/lib64/python2.7/subprocess.py", line 219, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
CalledProcessError: Command '['/home/gugl/dev/day0React/ardana-service/tools/mock-cmd', '/home/gugl/dev/day0React/ardana-service/data/config-processor/bin/python', '/home/gugl/dev/day0React/ardana-service/data/config-processor/ardana-configuration-processor/Driver/ardana-cp', '-l', '/tmp/tmpFWKCXd/log', '-c', '/home/gugl/dev/day0React/ardana-service/data/my_cloud/model/cloudConfig.yml', '-s', '/home/gugl/dev/day0React/ardana-service/data/ardana-input-model/2.0/services', '-r', '/home/gugl/dev/day0React/ardana-service/data/config-processor/ardana-configuration-processor/Data/Site', '-o', '/tmp/tmpFWKCXd/clouds']' returned non-zero exit status 254
INFO:ardana_service:2017-11-28 18:56:08.432145 127.0.0.1 400 POST http://localhost:8081/api/v2/config_processor

error from UI

POST http://localhost:8081/api/v1/clm/config_processor 400 (BAD REQUEST)
index.js:115197 Uncaught (in promise) TypeError: resp.json is not a function
    at index.js:115197
    at <anonymous>

Reply:
I re-selected a different model...the error is gone. Not sure what exactly caused the previous error, maybe just a fluke